### ISSUE TITLE
Extra Softcoded Options

### ISSUE THREAD

**Issue by Yuta12342** [ShadowMario/FNF-PsychEngine#11674]
### What feature do you want to get added on the **base** engine?

I couldn't tell if this was suggested before, as I tried searching, but the requests seemed different or confusing to follow, so I am just making my own for the sake of making sure it's understood.

Similar to how Credits was softcoded into Mods, I think that a txt or ini file that could hold extra Options or Settings that could be used globally for mods would be useful, such as a Mechanics Toggle or something of the sort, so it doesn't force people to create extra LUA Menus, Keybinds, or force people to edit the script directly.

Basically, I am thinking of making global variables that Scripts can access, but can be toggled or changed via the actual Options Menu. 

If you need more clarification, let me know.

(Also, sorry if this was requested before.)

### To test your sight, and reliability, please select the option of what should NOT be requested.

Stage Editor, 6K+ support, and winning icons.

**Comment by frantastic24** [comment]
#6636
#8002

**Comment by Yuta12342** [comment]
Oh, I see. 

I still feel like it'd be worth implementing, due to the nature of the engine, at some point.

I see a bunch of potential uses for it.

**Comment by frantastic24** [comment]
Sure but this is still a duplicate
